lib/player.js

```diff
@@ -7340,10 +7340,20 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
   async addFont(name, url) {
     if ('fonts' in document && 'FontFace' in window ) {
       await document.fonts.ready;
-      if (!('values' in document.fonts)) {
+      if (!('entries' in document.fonts)) {
         return;
       }
-      for (const fontFace of document.fonts.values()) {
+      const fontFaceSetIteratorToArray = (target) => {
+        const iterable = target.entries();
+        const results = [];
+        let iterator = iterable.next();
+        while (iterator.done === false) {
+          results.push(iterator.value);
+          iterator = iterable.next();
+        }
+        return results;
+      };
+      for (const fontFace of fontFaceSetIteratorToArray(document.fonts)) {
```

Comment:
Why not use `Array.from(document.fonts)`?

Reply:
See: https://sidneyliebrand.io/blog/fixing-font-face-set-entries-not-iterable-in-firefox `Array.from` is not working...